templates.ts

```diff
@@ -38,23 +38,6 @@ export const base = (
   }
 
   * { box-sizing: border-box; margin: 0; padding: 0; margin-block-start: 0; margin-block-end: 0; }
-
-  h1, h2, h3 { font-weight: 300; }
-
-  body { display: flex; flex-direction: column; align-items: center; min-height: 100vh; }
```

Comment:
I tried this out but I can’t reproduce it. It seems that the browser won’t render the page at all until all CSS has been loaded because [CSS is a render-blocking resource](https://web.dev/defer-non-critical-css/). That link explains you can use
```html
<link rel="preload" href="main.css" as="style" onload="this.onload=null;this.rel='stylesheet'">
<noscript><link rel="stylesheet" href="main.css"></noscript>
```
to load `main.css` asynchronously.

I’ll make sure to inline the layout styles again though.

Reply:
Would you rather I do that in this PR or in a follow-up?

Reply:
Let's leave it to a follow-up. I _think_ we might want to leave `@font-face` decls here, just so that the browser can immediately fire a GET for the .woff, but everything else should be moved to .css